R/R/metaflow_client.R

```diff
@@ -0,0 +1,147 @@
+#' Instantiate Metaflow flow/run/step/task client  
+#' @description A R6 Class representing a MetaflowClient used to inspect flow/run/step/task artifacts.
+#' This is a factory class that provides convenience for creating Flow/Run/Step/Task Client objects. 
+#'
+#' @docType class
+#' @importFrom R6 R6Class
+#'
+#' @return Object of \code{\link{R6Class}} with fields/methods for introspection.
+#' @format \code{\link{R6Class}} object.
+#'
+
+#'
+#' @section Usage:
+#' \preformatted{
+#' client <- mf_flow$new()
```

Comment:
nit -> `mf_flow` vs `mf_client`. What do you think of just `client`?

Reply:
Good catch. `client` is better but we still have `flow_client` / `run_client` / 'step_client` / `task_client` so it does not solve the problem too much. I'm also a bit concerned that `client` may conflict in namespace with other popular packages or user's own R code